lib/server.js

```diff
@@ -100,7 +100,7 @@ class S3Server {
 
 export default function main() {
     let clusters = _config.clusters || 1;
-    if (process.env.S3BACKEND) {
+    if (process.env.S3BACKEND === 'mem') {
```

Comment:
wouldn't that line be enough for rel/1.1, rather than bringing the whole default commit ?